docs/build.sh

```diff
@@ -35,4 +35,4 @@ $mcss_path/documentation/python.py conf.py
 echo "------------------------------------------------------------------------"
 echo "Docs were successfully generated. Open the following link to view them:"
 echo
-echo "file://$(pwd)/../../habitat-sim/build/docs/habitat-api/index.html"
```

Comment:
I don't see any formative changes in the build script though? I don't think these urls will be valid...

Reply:
There will be separate PR for HSIM and HWebsite and it became valid.